Tutorial/GettingStartedGuide.md

```diff
@@ -0,0 +1,65 @@
+# Getting Started Guide  
+  
+Creating a rest api to expose model's predict functionality  
+  
+  
+## Table of Contents  
+1. [pre-requisites](#pre-requisites)   
+2. [predict](#predict)  
+3. [api](#api)  
+4. [build-container](#build-container)  
+5. [run-container](#run-container)  
+5. [test-container](#test-container)  
+  
+## pre-requisites  
+  
+1. trained model  
+2. [source-to-image (s2i)](https://github.com/openshift/source-to-image)  
+3. [Httpie](https://httpie.org/) (Required to run `make test`)   
+  
+## predict     
+ Create a predict function that takes a message object as function parameter containing data instances to predict against   
+and return list of predictions   
+
+   > refer to function `predict_iris_type` in `iris_classification/predict.py`
+  
```

Comment:
iris is multiclass unordered, which isnt something we can scan. So although its a nice simple example I do have some qualms about it.

Reply:
great point, there should be something similar and familiar example we could use. I'll find out